config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml

```diff
@@ -8,6 +8,7 @@ dashboard_groups:
     - sig-cluster-lifecycle-cluster-api-1.3
```

Comment:
should we remove some of the older release jobs? How come we are testing so many release branches?

Reply:
We have maintained release-1.3 since release-1.3 was migrated to use EKS Community clusters. 
Nonetheless, you are right, we should be dropping release-1.3 and port over release-1.4 onto EKS community cluster or drop release-1.4 altogether and maintain release-1.3 test jobs.

Reply:
@kranurag7 Are we planning on porting over release-1.4 test jobs onto EKS community cluster and dropping release-1.3 jobs altogether? Was that the planned move?

Reply:
Found the PR moving release-1.4 to EKS Community cluster. 
https://github.com/kubernetes/test-infra/pull/30365
So I think yes, we should be dropping the release-1.3 jobs from the set of test jobs being maintained.

Reply:
> @kranurag7 Are we planning on porting over release-1.4 test jobs onto EKS community cluster and dropping release-1.3 jobs altogether? Was that the planned move?

Nawaz, I just pushed a change where I missed a resource block in one of the jobs. 
The 1.4 jobs is tracked here https://github.com/kubernetes/test-infra/pull/30365, but you're already aware of that. :)